pandas/core/dtypes/common.py

```diff
@@ -1223,6 +1223,10 @@ def is_numeric_dtype(arr_or_dtype) -> bool:
     >>> is_numeric_dtype(np.array([], dtype=np.timedelta64))
     False
     """
+    try:
+        return arr_or_dtype.kind in "uifcb"
+    except AttributeError:
+        pass
```

Comment:
comment to the effect of "fastpath/not a dtype object"?

will this change how is_numeric_dtype treats EA dtypes?